File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter02/CSHARP/CircleArea.cs
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 02.-Metodología de la programación y desarrollo de software.
    Problema 07.Encontrar el area y el perimetro de un círculo.

    A=PI*r*r
    P = 2*PI*r
*/

using System;

class CircleArea{

    public static void Main(string[] args){ 
        float r=0F;          //Variable radio      
        float PI=3.1416F;    //Variable PI
        float A=0F;          //Variable Area
        float P=0F;          //Variable Perimetro

        System.Console.Write("Ingrese el radio de la circunferencia: ");
        r=float.Parse(Console.ReadLine());

        A=PI*r*r;
        P=2*PI*r;
        
        System.Console.Write("\nRadio: "+r+"\nArea: "+A+"\nPerimetro: "+P+"\n\n");
    }
}

File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter02/JAVASCRIPT/js/circleArea.js
const PI=3.1416;    //Variable PI

function calculate(){
    let radio;
    let A;
    let P;

    let idParagrafh=document.createElement("p");
    idParagrafh.setAttribute("id", "idParagrafh");

   if(document.getElementById("idBtnCalculate").value=="Calcular"){
        radio=document.getElementById("idRadio").value;
        A=PI*radio*radio;          //Variable Area
        P=2*PI*radio;          //Variable Perimetro
        
        idParagrafh.innerHTML+="Area: "+A+"\nPerimetro: "+P;
        document.getElementById("idSection").appendChild(idParagrafh);
        
        document.getElementById("idBtnCalculate").value="Limpiar";
    }else{
        document.getElementById("idRadio").value="";
        document.getElementById("idParagrafh").remove();
        document.getElementById("idBtnCalculate").value="Calcular";
    }
    
}
/**
float r=0;          //Variable radio      
    
    

    printf("Ingrese el radio de la circunferencia: ");
    scanf("%f", &r);

    

    printf("\nRadio: %f\nArea: %f\nPerimetro: %f\n\n", r, A, P); 








 */
